src/views/Home.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
+    <HelloWorld></HelloWorld>
+  </div>
+</template>
+
+<script>
+import HelloWorld from "@/components/HelloWorld.vue";
+export default {
+  components: {
+    HelloWorld
+  }
+  // data() {
+  //   return {
+  //     myJson: json
+  //   };
+  // }
+  // mounted() {
+  //   this.getData();
+  // },
+  // methods: {
+  //   getData() {
+  //     // vue-resource example
+  //     // const jsonlink = JSON.stringify("/salads.json");
+  //     // const link = JSON.parse(jsonlink);
+  //     debugger;
+  //     this.axios
+  //       .get("/salads.json")
+  //       .then(response => {
+  //         console.log(response.data);
+  //         this.data = JSON.parse(response.data);
+  //       })
+  //       .catch(e => {
+  //         console.log("Error", e);
+  //       });
+  //   }
+  // }
+};
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
+ul {
+  list-style-type: none;
+  padding: 0;
+}
+li {
+  display: inline-block;
+  margin: 0 10px;
+}
+</style>
```

Comment:
W zasadzie cały ten plik jest zbędny. Po pierwsze jedyne co robi to importuje i używa komponentu HelloWorld (btw. nazwa nie trafiona :D wg. powszechnych style guidów nazywaj komponenty na podstawie <domena>-<funkcjonalność> np. salad-list). Czyli ten komponent jest wraperem, czy dodaje jakąś funkcjonalność lub style? Nie. Nie ma funkcjonalności, a style są ale enkapsulowane (scoped), dlatego nie będą wykorzystane nigdzie indziej. Cały ten komponent można usunąć.